### DISCUSSION TITLE
What are the rules for discussions?

### DISCUSSION THREAD

**Discussion by ippolite** [daveshap/ACE_Framework#12] [General]
I can't find a single and comprehensive set of rules for the discussions here.
It'd be great if we together come up with some guidelines for each section. Say, a few templates for teams (hiring/looking for), some bullet points for demos and implementations, and, of course, how to ask a great question in the Q&A section.

**Comment by daveshap** [comment]
https://github.com/daveshap/ACE_Framework/blob/main/contributing.md#contributing-to-ace_framework

**Reply by ippolite** [reply]
Oh, the C3P0 document! I surely forgot about it.

I also believe that some templates would do a fair bit in the long run. I think I will propose some to the general public and we see what sticks.

**Reply by daveshap** [reply]
No, let's not go overboard. Thanks though
